listener/avalon_listener/tcs_work_order_handler.py

```diff
@@ -172,14 +184,19 @@ def WorkOrderSubmit(self, **params):
         wo_id = params["workOrderId"]
         input_json_str = params["raw"]
         input_value_json = json.loads(input_json_str)
-
+        # Work order status payload should have
+        # data filed with work order id as defined in EEA spec section 6.
+        data = {
+            "workOrderId": wo_id
+        }
         req_validator = WorkOrderRequestValidator()
         valid, err_msg = req_validator.validate_parameters(
             input_value_json["params"])
         if not valid:
             raise JSONRPCDispatchException(
                 JsonRpcErrorCode.INVALID_PARAMETER,
-                err_msg
+                err_msg,
+                data=data
```

Comment:
Any specific reason why you used data=data instead of just passing data.

Reply:
JSONRPCDispatchException has default arguments and variable arguments, it is named argument to that function.